src/util/GlobalEvents.ts

```diff
@@ -58,7 +58,7 @@ export namespace GlobalEvents {
         if (callback(event)) return true;
       }
 
-      return false;
+      return true;
```

Comment:
Should we not just delete the for loop here since the trigger always returns true?